.travis.yml

```diff
@@ -1,10 +1,12 @@
 ---
 language: python
-python: "2.7.15"
+python: "3.9"
 sudo: required
 
 env:
   - PIPENV_IGNORE_VIRTUALENVS=1
```

Comment:
I think this line is no longer useful, because it triggers a "false" build

![image](https://github.com/idealista/opengrok_role/assets/23662639/d235bac4-4336-4974-ba68-a6111bb53e00)